### PR TITLE
GCP VM Disk Loss Experiment Docs

### PR DESCRIPTION
**Pull request by neelanjan00**
Added GCP VM Disk Loss experiment docs and sidebar id

### CODE REVIEW

File: docs/gcp-vm-disk-loss.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+  </tr>
+  <tr> 
+    <td> TOTAL_CHAOS_DURATION </td>
+    <td> The time duration for chaos insertion (sec) </td>
+    <td> Optional </td>
+    <td> Defaults to 30s </td>
+  </tr>
+  <tr> 
+    <td> CHAOS_INTERVAL </td>
+    <td> The time duration between the attachment and detachment of the volumes (sec) </td>
```
Comment (ispeakc0de) [inline docs/gcp-vm-disk-loss.md]: It is the time interval between successive chaos interations

Comment (neelanjan00) [reply docs/gcp-vm-disk-loss.md]: Fixed

File: website/sidebars.json [hunk 2]
```diff
@@ -82,6 +82,7 @@
         "type": "subcategory",
         "label": "GCP",
         "ids": [
+          "gcp-vm-disk-loss"
```
Comment (ispeakc0de) [inline website/sidebars.json:86]: Can you add the comma here?

Comment (neelanjan00) [reply website/sidebars.json:86]: Fixed

**Review by ispeakc0de — DISMISSED** [review]

File: docs/gcp-vm-disk-loss.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    <td> Optional </td>
+    <td> Defaults to 30s </td>
+  </tr>
+  <tr>
+    <td> SEQUENCE </td>
+    <td> It defines sequence of chaos execution for multiple instance</td>
+    <td> Optional </td>
+    <td> Default value: parallel. Supported: serial, parallel </td>
+  </tr> 
+  <tr>
```
Comment (uditgaurav) [inline docs/gcp-vm-disk-loss.md:189]: Ramp time ENV is missing.

File: docs/gcp-vm-disk-loss.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+- If the chaos experiment is not executed, refer to the [troubleshooting](https://docs.litmuschaos.io/docs/faq-troubleshooting/) section to identify the root cause and fix the issues.
+
+### Watch Chaos progress
+  
+- Monitor the attachment status for ebs volume from AWS CLI.
+
+  `gcloud compute disks describe DISK_NAME --zone=DISK_ZONE`
+
+- GCP console can also be used to monitor the disk volume attachment status.   
+
```
Comment (uditgaurav) [inline docs/gcp-vm-disk-loss.md:271]: abort section is missing.

File: docs/gcp-vm-disk-loss.md (hunk 3 above)
Comment (neelanjan00) [reply docs/gcp-vm-disk-loss.md:189]: fixed

File: docs/gcp-vm-disk-loss.md (hunk 4 above)
Comment (neelanjan00) [reply docs/gcp-vm-disk-loss.md:271]: fixed

**Review by ispeakc0de — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**